ci/cpu/upload.sh

```diff
@@ -14,6 +14,9 @@ export GPUCI_RETRY_SLEEP=30
 # Set default label options if they are not defined elsewhere
 export LABEL_OPTION=${LABEL_OPTION:-"--label main"}
 
+# Setup conda build dir
+export CONDA_BLD_DIR="$WORKSPACE/.conda-bld"
+
```

Comment:
```suggestion
```

These lines can be removed. Since `CONDA_BLD_DIR` is `export`ed in the parent `ci/cpu/build.sh` script, it shouldn't be necessary to redefine it here.